docs/reference/reana-yaml/index.md

```diff
@@ -113,6 +113,7 @@ The **workflow** property is composed of:
 | type          | _string_     | _mandatory_                                        | Specifies workflow language type. Can be `cwl`, `serial`, `yadage`, `snakemake`.                                                             |
 | file          | _string_     | _mandatory if property `specification` is missing_ | For CWL, Yadage and Snakemake workflows, specifies workflow steps in an external file, using their respective workflow definition languages. |
 | specification | _dictionary_ | _mandatory if property `file` is missing_          | For Serial workflows, specifies workflow steps internally in `reana.yaml`, see below.                                                        |
+| resources     | _dictionary_ | _optional_                                         | Specifies the resources needed by the workflow, see below.                                                                                   |
```

Comment:
Specifies additional resources...

(or "extra resources"?)

Reply:
Changed it to _additional resources_